client/extensions/woocommerce/state/ui/payments/methods/test/reducer.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer, { initialState } from '../reducer';
+import {
+	openPaymentMethodForEdit,
+	closeEditingPaymentMethod,
+	cancelEditingPaymentMethod,
+	changePaymentMethodField,
+} from '../actions';
+
+const siteId = 123;
+
+describe( 'reducer', () => {
+	describe( 'openPaymentMethodForEdit', () => {
+		it( 'when the method has a server-side ID', () => {
```

Comment:
Oh my... I've been using this format in most of my tests :(

The reason was that, when you run the tests, you get "not-so-weird" sentences, like:
```
- reducer
  - getSomething
    ✔️  when there is nothing
    ✔️  when there is already something
```